specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2020-07-15/examples/GetIdentityInfo.json

```diff
@@ -0,0 +1,13 @@
+{
+  "parameters": {
+    "api-version": "2020-07-15",
```

Comment:
@rifrankl This PR contains multiple API versions which makes it difficult to review. I would recommend to raise a separate PR for each version.

While sending a PR for new api version, the recommendation is to copy the existing version into new directory structure for first commit and then push new changes in subsequent commits. This helps in easily identifying the delta changes from previous versions.

Please raise new PRS with the above recommendations for faster review.